pkg/dockermanager/dockermanager.go

```diff
@@ -123,7 +123,9 @@ func (dm *Manager) Manage(
 	defer cancel()
 
 	stop := func(container string) {
+		dm.S().Infow("got stop event", "container", container)
 		if m, ok := managers[container]; ok {
+			dm.S().Infow("calling m.cancel()", "container", container)
 			m.cancel()
```

Comment:
This is currently not working, for some reason. Looks like we don't propagate contexts correctly.